src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java

```diff
@@ -695,4 +695,29 @@ public void testConfigWithIgnoreExceptionalAttributes() {
         }
     }
 
+    @Test
+    public void testLoadConfiguration3() throws Exception {
+        final String[] configFiles = {
+            "InputConfigurationLoaderOldConfig0.xml",
+            "InputConfigurationLoaderOldConfig1.xml",
+            "InputConfigurationLoaderOldConfig2.xml",
+            "InputConfigurationLoaderOldConfig3.xml",
+            "InputConfigurationLoaderOldConfig4.xml",
+            "InputConfigurationLoaderOldConfig5.xml",
+            "InputConfigurationLoaderOldConfig6.xml",
+            "InputConfigurationLoaderOldConfig7.xml",
+        };
+
+        for (String configFile : configFiles) {
+            final DefaultConfiguration config =
+                    (DefaultConfiguration) ConfigurationLoader.loadConfiguration(
+                            new InputSource(Files.newInputStream(Paths.get(
+                                    getPath(configFile)))),
+                            new PropertiesExpander(new Properties()),
+                            IgnoredModulesOptions.OMIT);
+            assertWithMessage("should not have children")
+                    .that(config.getChildren())
+                    .isEmpty();
```

Comment:
Should not be empty.....

Strange that it is empty , please look for a reason.
Confiy should be loadable and produce violation if required.

Can we regular verify test method execution?

Reply:
@romani If i am not wrong all the content of all the xml contains checker only 

```
<module name="Checker">
</module>
```
if the file is like 
```
<module name="Checker">
    <module name="TreeWalker">
        
    </module>
</module>

```
then the child is not empty, because it is treewalker

> Can we regular verify test method execution? 

for this I have to extend AbstractModuleTestSupport and this test class already extends AbstractPathTestSupport

Reply:
we can skip regular verify method execution here, but we need ^^^^

Reply:
Done please check